packages/cli/src/commands/doctor/doctor.js

```diff
@@ -0,0 +1,131 @@
+import chalk from 'chalk';
+import envinfo from 'envinfo';
+import {logger} from '@react-native-community/cli-tools';
+import ISSUES from './issues';
```

Comment:
Reading through these changes, I struggled with the term "issue" because until these are checked there is nothing wrong (i.e. the issues are not issues)

What do you think of calling these something like healthchecks?

Reply:
I also struggled to name these, definitely changing it to `healthchecks`!

Reply:
I also struggled to name these, definitely changing it to healthchecks!